core/controllers/reader.py

```diff
@@ -332,9 +332,6 @@ def post(self, unused_exploration_id):
         old_state = exp_domain.State.from_dict(self.payload.get('old_state'))
         # The learner's raw answer.
         answer = self.payload.get('answer')
-        # The learner's parameter values.
```

Comment:
OK. (Surprised that it didn't get picked up before, but you're right -- thanks for explaining.)